docs/installation/production/installation.rst

```diff
@@ -9,13 +9,82 @@ This article will provide an overview of how to install Tethys Portal in a produ
 1. Install Tethys Portal
 ========================
 
-Follow the default :doc:`../../installation` instructions to install Tethys Portal with the following considerations
+Follow these steps to install Tethys Portal with the following considerations
 
 * Make sure to install the correct version.
 * Assign strong passwords to the database users.
 * You must edit the :file:`portal_config.yml` file to ensure production settings (i.e. ``DEBUG: False``, etc.)
 * Optionally, Follow the :doc:`./distributed` instructions to install Docker and the components of the software suite on separate servers.
 
+a) Install with Conda
+---------------------
+
+To install the ``tethysplatform`` into a new conda environment then run the following commands::
+
+    conda create -n tethys -c tethysplatform -c conda-forge tethysplatform
+    conda activate tethys
+
+Some of the following steps are dependent on the specific distribution where ``tethysplatform`` is installed.
+
+* Install Supervisor (See `<http://supervisord.org/installing.html>`_)
+* Install NGINX (See `<https://www.nginx.com/resources/wiki/start/topics/tutorials/install/>`_)
+* Generate Tethys portal configuration:
+
+    .. code-block::
+
+        $ tethys gen portal_config
+
+        $ tethys settings --set ALLOWED_HOSTS "<ALLOWED_HOST>" --set DATABASES.default.USER <TETHYS_DB_USERNAME> --set DATABASES.default.PASSWORD <TETHYS_DB_PASSWORD> --set DATABASES.default.PORT <TETHYS_DB_PORT> --set DATABASES.default.DIR <TETHYS_DB_DIR>
+
+* Setup Tethys database:
+
+    .. code-block::
+
+        $ tethys db configure --username <TETHYS_DB_USERNAME> --password <TETHYS_DB_PASSWORD> --superuser-name <TETHYS_DB_SUPER_USERNAME> --superuser-password <TETHYS_DB_SUPER_PASSWORD> --portal-superuser-name <TETHYS_SUPER_USER> --portal-superuser-email '<TETHYS_SUPER_USER_EMAIL>' --portal-superuser-pass <TETHYS_SUPER_USER_PASS>
+
+* Install Docker (See `<https://docs.docker.com/v17.09/engine/installation/>`_)
```

Comment:
Move install Docker step up with the other system install steps after NGINX and Supervisor. Also indicate that this is optional.